build/project-template-gradle/build.gradle

```diff
@@ -270,6 +270,17 @@ copyMetadata.dependsOn(buildMetadata)
 
 task buildapk {
 	
+	  task includePluginBuildFiles {
+
+      new File("$nodeModulesDir").eachFileRecurse(groovy.io.FileType.FILES) {
```

Comment:
Good point. As I think through this do you believe it will have a performance impact if we have many modules, some of them even as `devDependencies`? Wouldn't it be better if the CLI does something in the `plugin prepare` phase (executed on `plugin add`)?